web/developer.actyx.com/docs/conceptual/how-actyx-works.mdx

```diff
@@ -22,15 +22,19 @@ Traditionally, you would write two apps, one dealing with each asset. These apps
 
 ### The local-twin programming model
 
-With the Actyx Platform, you program the process as autonomous _local twins_ that publish and consume events, and develop states based on these events:
+With the Actyx Platform, you program the process as autonomous _local twins_, that is a _digital twin_ that reside, perceive, and act locally.
+These _local twins_ publish and consume events, and develop states based on these events:
 
 ![how-actyx-works-3](/images/conceptual/how-actyx-works/how-actyx-works-3.png)
 
-Local twins are implemented in the Typescript programming language using APIs provided by the **Actyx Pond** library.
+There are two ways in which the business logic can be formulated:
+
+-
+
 There are two different ways in which the business logic can be formulated:
 
-- If the local twin models something where your application shall control or coordinate the progression through a sequence of different life cycle states then you use **fishes**.
-- Otherwise you can use stateless logic, emit events as they occur in the real world, and retrieve information using the Actyx Query Language.
+- You can use stateless logic; using the SDK, emit events as they occur in the real world and retrieve information using the Actyx Query Language
+- Otherwise, if the local twin models something where your application shall control or coordinate the progression through a sequence of different life cycle states than you can use [`machine-runner`](https://www.npmjs.com/package/@actyx/machine-runner) which the usage of can be found in the library documentation.
```

Comment:
I’d close with “… you can use machine-runner; please find its docs in the library documentation”.